device/demo/actors/player/player.tscn

```diff
@@ -164,4 +173,8 @@ anims/walk_right = SubResource( 6 )
 anims/walk_up = SubResource( 7 )
 blend_times = [  ]
 
+[node name="CollisionShape2D" type="CollisionShape2D" parent="." index="2"]
+
+shape = SubResource( 8 )
+
```

Comment:
These changes will require some documentation as well.

Reply:
Added it, will force-push soon